integration-tests/test_stdin_to_stdout.py

```diff
@@ -4,7 +4,7 @@
 from common.utils import run_job
 
 
-def test_stdin_to_stdout(tmpdir: str):
```

Comment:
any reason this was changed?

Reply:
yes, it's not a str, but a pytest object and it was confusing to see read() on str.